components/paddock/paddock/settings.py

```diff
@@ -115,26 +121,35 @@
 # Database
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
-# Parse database connection url strings
-# like psql://user:pass@127.0.0.1:8458/db
 if "test" in sys.argv or "test_coverage" in sys.argv:
     DATABASES = {
         "default": {
-            "ENGINE": "django.db.backends.sqlite3",
-            "NAME": BASE_DIR / "db.sqlite3",
+            "ENGINE": "django_prometheus.db.backends.sqlite3",
+            "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
         },
         "readonly": {
-            "ENGINE": "django.db.backends.sqlite3",
-            "NAME": BASE_DIR / "db.sqlite3",
+            "ENGINE": "django_prometheus.db.backends.sqlite3",
+            "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
         },
     }
 else:
     DATABASES = {
-        # read os.environ['DATABASE_URL'] and raises
-        # ImproperlyConfigured exception if not found
-        #
-        "default": env.db_url("DATABASE_URL"),
-        "readonly": env.db_url("READONLY_DATABASE_URL"),
+        "default": {
+            "ENGINE": "django_prometheus.db.backends.postgresql",
+            "NAME": os.getenv("DB_NAME", "paddock"),
+            "USER": os.getenv("DB_USER", ""),
+            "PASSWORD": os.getenv("DB_PASSWORD", ""),
+            "HOST": os.getenv("DB_HOST", ""),
+            "PORT": os.getenv("DB_PORT", "5432"),
+        },
+        "readonly": {
```

Comment:
@durandom I am unsure how this gets created, not via https://github.com/b4mad/racing/blob/main/manifests/base/postgresql/cluster.yaml#L82